### ISSUE TITLE
[1.16.5 - Forge: 36.2.39] Mouse Tweaks / Pixelmon functionality conflicts

### ISSUE THREAD

**Issue by Voidion** [YaLTeR/MouseTweaks#89]
In MC 1.16.5 - Mouse Tweaks version `2.14-mc.1.16.2`, when combined with the latest versions of Pixelmon for 1.16.5 so versions `9.0.0` and up the game refuses to allow single item split stacking, mouse dragging, among other things that are added by the base game and MouseTweaks. 

In doing further research on my own with this issue, I've found a very interesting edge case, if you go into JEI and look at a recipe, and then go back into the inventory GUI, the functionality is fully restored. But the functionality is lost upon exiting the inventory GUI back into the main game and, returning to the inventory GUI the functionality is lost again until you do the recipe tab trick.

All other GUI's except for the inventory GUI work as intended, so crafting, chests, furnaces, etc. Pixelmon does edit the main inventory GUI, but something about it is causing these issues. And the part that doesn't make sense to me is that as I said, if you do the recipe trick it gets restored until exiting.

**Comment by YaLTeR** [comment]
Thanks for the detailed report! Have you reported it to Pixelmon? Kinda sounds like it might be something on their side.

**Comment by Voidion** [comment]
Yeah I have, and hopefully they'll get around to looking into it. One of the reasons I reported it here though is that weird edge case I mentioned but also in case it was on Mouse Tweaks end instead of Pixelmon's end. Also thanks for the reply :)

**Comment by YaLTeR** [comment]
Do you mind linking the Pixelmon issue here?

**Comment by Voidion** [comment]
https://pixelmonmod.com/tracker.php?p=2&t=18224 hopefully this is what you meant, this is from their issue tracker
